opensearch-persistence/spec/repository/find_spec.rb

```diff
@@ -46,17 +46,6 @@
         expect(repository.exists?('1')).to be(false)
       end
     end
-
-    context 'when options are provided' do
```

Comment:
The option that was provided was the type. However, type is deprecated hence the test case is removed